bcda/api/v1/api.go

```diff
@@ -22,8 +27,11 @@ import (
 )
 
 var h *api.Handler
+var repository *postgres.Repository
 
 func init() {
+	repository = postgres.NewRepository(database.Connection)
```

Comment:
It looks like this is already available through the `h.Svc` variable. You may have to add to it's set of functions.